vendor.mod

```diff
@@ -51,7 +51,7 @@ require (
 	github.com/klauspost/compress v1.15.9
 	github.com/miekg/dns v1.1.43
 	github.com/mistifyio/go-zfs v2.1.2-0.20190413222219-f784269be439+incompatible
-	github.com/moby/buildkit v0.10.5
+	github.com/moby/buildkit v0.10.6
```

Comment:
Can we update `x/crypto` too for `docker build --ssh`?
- https://github.com/moby/buildkit/pull/3274

Reply:
Will open a PR

Reply:
- https://github.com/docker/buildx/pull/1407
- https://github.com/docker/cli/pull/3862